task/redhat-codeready-dependency-analysis/0.1/samples/pipeline-run.yaml

```diff
@@ -0,0 +1,29 @@
+# PipelineRun for starting pipeline
+# Make sure Workspace is attached and Secret is created,
+# also configure 'pre-task-setup' and 'post-task-setup' tasks before running the pipeline.
+# refer examples from samples
+
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: redhat-codeready-dependency-analysis-pipeline
+spec:
+  pipelineRef:
+    name: redhat-codeready-dependency-analysis-pipeline
+  workspaces:
+    - name: output
+      persistentvolumeclaim:
+        claimName: output
+
+# Change link to git_url and manifest-file-name according to target repository
+  params:
+    - name: file_url
+      value: {{MANIFEST_FILE_URL}} # Replace it with URL of manifest file to be scanned for demo.
```

Comment:
I am not sure I understand that comment,

Reply:
This is for showing a demo of this task in an e2e pipeline, here this demo pipeline takes a manifest file from a given URL and shows analysis of dependencies present in file.
Here is an example: 
[https://raw.githubusercontent.com/<REPO_NAME>/master/requirements.txt](https://raw.githubusercontent.com/<REPO_NAME>/master/requirements.txt)

Users are responsible for placing the target manifest file into workspace, it means he/she can download it from a URL as given in example, or create/copy it by another task or clone project into workspace using git-clone task from Tekton Hub etc.